src/graphql/resolvers.ts

```diff
@@ -0,0 +1,71 @@
+import { Resolvers, TeamMvc } from './types';
+import { connect } from '../dao';
+import { TeamMvcDbObject } from '../dao/types';
+import { ObjectID } from 'mongodb';
+
+const dbPromise = connect();
+
+const getCollection = async () => {
+  const db = await dbPromise;
+  return db.collection<TeamMvcDbObject>('teams');
+};
+
+const fromDbObject = (dbObject: TeamMvcDbObject): TeamMvc => ({
+  team_id: dbObject.team_id,
+  name: dbObject.name,
+  auction_budget_total: dbObject.auction_budget_total,
+  auction_budget_spent: dbObject.auction_budget_spent,
+  wavier_priority: dbObject.wavier_priority,
+  faab_balance: dbObject.faab_balance,
+  number_of_moves: dbObject.number_of_moves,
+  number_of_trades: dbObject.number_of_trades,
+  managers: dbObject.managers,
+  team_logos: dbObject.team_logos
+});
+const resolvers: Resolvers = {
+  Query: {
+    allTeams: async () => {
+      const collection = await getCollection();
+      return await collection.find().map(fromDbObject).toArray();
+    },
+    Team: async (_: any, { team_id }) => {
+      const collection = await getCollection();
+      const dbObject = await collection.findOne({
+        _id: ObjectID.createFromHexString(team_id)
+      });
+      console.log(dbObject);
```

Comment:
clean up console.log

Reply:
cleaned up will be a part of next commit